app/mailers/startup_mailer.rb

```diff
@@ -51,4 +51,23 @@ def grading_details(startup_feedback, include_grades)
       ).html_safe
     end
   end
+
+
+  def comment_on_submission(submission, comment, user)
+    @submission = submission
+    @students = submission.students.map(&:fullname).join(", ")
```

Comment:
Let's differentiate from instances of the model.

```suggestion
    @student_names = submission.students.map(&:fullname).join(", ")
```